windows/windows_update_install/prepare_msu_file.yml

```diff
@@ -24,6 +24,14 @@
 
 - name: "Get unused driver letter"
   include_tasks: ../utils/win_get_unused_drive_letter.yml
+  when: guest_os_build_num | int >= 17763
+
+# For OS older than 17763 (Windows 1809 or Windows Server 2019), it has very high frequency of timeout
```

Comment:
For OS older than 17763 -> For OS build number older than 17763
Windows 1809 -> Windows 10, v1809

Reply:
Updated. Thanks.